src/apps/activities/crud/activity.py

```diff
@@ -55,9 +58,44 @@ async def get_mobile_with_items_by_applet_id(self, applet_id: uuid.UUID, is_revi
             ActivitySchema.order,
             ActivitySchema.scores_and_reports,
         )
+
+        # Query filters
+        filters = query_params.filters
+
+        if filters.get("has_score") is not None:
+            query = query.join(
```

Comment:
You cannot merge answers and answer_items tables to other ones because of arbitrary server.

Reply:
This is joining the `activity_items` so I can check if `score` is set for this activity

Reply:
Sorry, missclick, wrong line